src/components/Game/Game.js

```diff
@@ -1,35 +1,43 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 
 import classes from './Game.scss';
 import { Slots } from './Slots/Slots';
 import { Letters } from './Letters/Letters';
+import Result from './Result/Result';
 
 import { WORD_LENGTH } from '../../constants/constants';
 import { shuffleArray, generateID } from '../../helpers';
-import { getCorrectWords } from '../../service/service';
-
+import { getCorrectWordsAction } from '../../store/actions';
 
 class Game extends Component {
     state = {
-        correctWords: [],
         letters: [],
         lettersInSlots: [],
         gameWon: undefined,
     };
 
     componentDidMount() {
-        const correctWords = getCorrectWords();
-        const lettersObjects = correctWords[0].split('')
-            .map((letter) => ({
-                value: letter,
-                id: generateID(),
-                active: true,
-            }));
+        const { getCorrectWords } = this.props;
+        getCorrectWords();
```

Comment:
this function return an action payload
as mentioned elsewhere this action payload should include correct words array
so actually you can initialize `state.letters` here directly without `componentDidUpdate`

Reply:
hmm, isn't it dispatching action only? I didn't manage to initialize `letters` here, neither now, nor before. I kept getting errors since `correctWords` was still empty in this lifecycle. hence `componentDidUpdate`. I don't know, maybe I'm missing sth.

Reply:
please run the code `console.log(getCorrectWords())`;

Reply:
> `correctWords` was still empty in this lifecycle
I believe you can get `correctWords` when you dispatch the action as dispatch has a return value

Reply:
I know what this function does ;) but it's not my point exactly. I thought `correctWords` needs to come from the store (so: mounting -> action -> getting data from store). if I understand you correctly now it's not a must(?). and what you suggest is getting words by simply invoking `getCorrectWords()` (plus passing it to store), so that it would be pretty the same what I did in `componentDidMount` before store setup. is this correct?

Reply:
> I thought correctWords needs to come from the store

yeah, that would be perfect… store serving as a source of truth in the app
I suggested using return value of `getCorrectWords` to simplify the code but perhaps you're right and this solution is not elegant, it's a shortcut to some extent, subtle one, like saying to the store "I know what you're going to say in a moment!"

so there are two solutions: using return value of `getCorrectWords` (quick) or `componentDidUpdate` approach (more "correct"), I'm okay with both!

> so that it would be pretty the same what I did in componentDidMount before store setup

except we have an advantage of having store now and being able to use `correctWords` on Result route (I imagine having at least three routes: Home, Game, Result - as suggested on Trello board)
also, it would be perfectly possible to not use redux store in this project as it's a rather simple app but implementing it serves educational purposes, I recall you wanted to practice redux store

Reply:
sorry for the confusion! :)